sources/dfmc/reader/tests/literal-test-suite.dylan

```diff
@@ -19,13 +19,15 @@ define function verify-presentation
 end function;
 
 define test binary-integer-literal-test ()
+/*
   verify-literal(read-fragment("#b10"), 2, <integer-fragment>);
   verify-literal(read-fragment("#b010"), 2, <integer-fragment>);
   verify-literal(read-fragment("#b1111_0000"), 240, <integer-fragment>);
+*/
   assert-false(read-fragment("#b10_"));
-  assert-signals(<invalid-token>, read-fragment("#b_10"));
-  assert-signals(<invalid-token>, read-fragment("#b2"));
-  assert-signals(<invalid-token>, read-fragment("#b + 1"));
+//  assert-signals(<invalid-token>, read-fragment("#b_10"));
```

Comment:
Why are these commented out?

Reply:
Fixed by uncommenting them. Looks like something I broke at one point and was debugging by commenting out all but one assertion. Thanks.